### ISSUE TITLE
Not working for Epic Games version...

### ISSUE THREAD

**Issue by norrin77** [Nibre/MotherVR#201]
which is just 1.0RC5.284847.51533

**Comment by shreywy** [comment]
It shows me this error message when I launch from epic, using the newest beta version. 
![image](https://user-images.githubusercontent.com/76450209/102812150-90e3c880-4394-11eb-9cdc-7290933f6e8a.png)

The directory is my HDD Epic games file, not my steam if that is important

**Comment by Nibre** [comment]
The check is failing because the version on Epic is a newer build of the game, versus the one on Steam which hasn't been updated since 2015.

Downloading it now to see how similar they are, and to check if a simple tweak on my end should be good enough to get it working in the mean time. Worst case I'll just have to patch some offsets to align for the Epic version instead, until I can backport some of my newer dynamic ReclaimerVR code in the next version of MotherVR.

**Comment by battleboy321** [comment]
Do you know how much is gonna take you to do it I don't want to be a bad person just really want to play it and btw thanks for doing it for us for the community

**Comment by Troubadour42** [comment]
I have the same issue too with the Epic version so I'm hoping Nibre can do the fix!  No pressure!  :-D

**Comment by PvtCowboy** [comment]
Spent 4 hours trying to get the mod to working with the pc game pass version last night. Gave up as I couldn't work around the permissions Microsoft Game Pass enforce on the game... today free on Epic... sweet, and the patch doesn't work, but looks like Nibre is on it like sonic, so much appreciated!

**Comment by sassi006** [comment]
> The check is failing because the version on Epic is a newer build of the game, versus the one on Steam which hasn't been updated since 2015.
> 
> Downloading it now to see how similar they are, and to check if a simple tweak on my end should be good enough to get it working in the mean time. Worst case I'll just have to patch some offsets to align for the Epic version instead, until I can backport some of my newer dynamic ReclaimerVR code in the next version of MotherVR.

First, I want to say thanks for your work here. Would it at all be possible for you to make this code open source though? I've been wanting to play this for over a year or two now, but have been unable to due to a gamebreaking bug. If you made this open source, then the community could help make it better and the hard work you put into this could continue to progress.

**Comment by bdolan1125** [comment]
Just here to chime in with a THANK YOU! for your immediate attention to us, the fans! YOU ROCK! ;D

**Comment by syclonix** [comment]
I also wanted to drop by and say a BIG THANK YOU for looking into supporting the Epic version of the game. 

Merry Christmas and Happy New Year to you Nibre!

**Comment by djstatik** [comment]
Thanks for looking into this Nibre!  Should be a lot more players using this mod, if you can support the Epic Games version.  Money coming your way when the patch is ready!

**Comment by Deadtenor** [comment]
Thanks to the dev for this project.  Hoping to see some love for the Epic Games Store version.

**Comment by asxuri** [comment]
Thanks! another Epic Game player with same bug

**Comment by rstoop** [comment]
Yes thanks man for looking into it. Will be watching this thread.

**Comment by Steve-XM** [comment]
Hello,

Just another person who picked up the game for free on the Epic Store. Looking forward to seeing the Epic Store fix for the Mother VR Mod. Keep up the good work man. We all owe you a beer :) I can confirm that I get the same error as @sk8gamer . I am going to be playing on a Rift S if that helps at all. Willing to provide feedback.

**Comment by WKampel** [comment]
I would pay big money to have this mod work flawlessly, and I'm sure many other people would agree.

**Comment by gattocerber** [comment]
ditto like all of the above, would love to try the Epic version out.

**Comment by ghost** [comment]
will use on oculus quest thru link when ready! am excited to try the game out for the first time ever on vr!

**Comment by ricardokung** [comment]
.Thank you Nibre for looking into this.

**Comment by wrexinite** [comment]
I just realized this repo only contains the README, lol.  I kinda assumed the real source would be up. :D
If ya want a hand hmu

**Comment by Nibre** [comment]
I've now mapped out and adjusted all of the offsets (100+) to match where they are in the Epic Store version, but it looks like the menus have also been modified a bit, so that means there's a little more work I'll need to do to get everything functioning correctly.

**Comment by thundercato** [comment]
> I've now mapped out and adjusted all of the offsets (100+) to match where they are in the Epic Store version, but it looks like the menus have also been modified a bit, so that means there's a little more work I'll need to do to get everything functioning correctly.

Thanks Nibre, I think it's clear to see by all the comments that your time and effort are appretiated by all. How do you actually fix it? Are you using a hex editor, a Mod? it's way above my 6802 and QBasic programming head lol

**Comment by ghost** [comment]
> 
> 
> I've now mapped out and adjusted all of the offsets (100+) to match where they are in the Epic Store version, but it looks like the menus have also been modified a bit, so that means there's a little more work I'll need to do to get everything functioning correctly.

Thank you! We appreciate it.
Good luck!

**Comment by NeilZander** [comment]
It's exciting to hear from Nibre after all this time hope your doing well! And looking forward to the  results of this working with the Epic Games Store version and any other hopeful future updates you have in store! Good Luck and Thank you for everything you've done!

**Comment by synthercat** [comment]
I'll be waiting for this!

**Comment by Alan-Cha** [comment]
I am also having the same problem. Glad to hear that @Nibre is taking a look at this. Thank you for the continued support!

**Comment by DocJellyBean** [comment]
> 
> 
> I've now mapped out and adjusted all of the offsets (100+) to match where they are in the Epic Store version, but it looks like the menus have also been modified a bit, so that means there's a little more work I'll need to do to get everything functioning correctly.

Looking forward to it so that I can try my new VR hesdset. Many thanks for looking to update it.

**Comment by Kruxis** [comment]
Hey man! @Nibre 

I am very glad that there is a VR mod for the game and i am totally hyped for a functioning version for the Epic Games version!

I don't want to preasure at all but is there an ETA off when it will be ready? =)

**Comment by MartinNuc** [comment]
Guys! It's Christmas!! Give him some rest! This is not how you should treat a person who does it in his spare time.

When the patch is ready he is going to share it with us. No need to post here anything. Just be patient.

**Comment by JoeyDagger** [comment]
> 
> 
> Guys! It's Christmas!! Give him some rest! This is not how you should treat a person who does it in his spare time.
> 
> When the patch is ready he is going to share it with us. No need to post here anything. Just be patient.

The hell are you talking about? Nobody is treating him in any ill way. Everyone is saying thank you to him for working on it and taking time out of his day.

**Comment by ghost** [comment]
> 
> 
> Hey man! @Nibre
> 
> I am very glad that there is a VR mod for the game and i am totally hyped for a functioning version for the Epic Games version!
> 
> I don't want to preasure at all but is there an ETA off when it will be ready? =)

When it's ready.

**Comment by AriesKato** [comment]
Since we all got the Epic game version for free...
Why wouldn't we all encourage Nibre with some nice festive beer or two?

[Nibre's Paypal](https://www.paypal.me/NibreVR)

Merry Christmas and thanks for the hard work!

**Comment by DocJellyBean** [comment]
> 
> 
> Since we all got the Epic game version for free...
> Why wouldn't we all encourage Nibre with some nice festive beer or two?
> 
> [Nibre's Paypal](https://www.paypal.me/NibreVR)
> 
> Merry Christmas and thanks for the hard work!

Good idea, done!

**Comment by thundercato** [comment]
> > Since we all got the Epic game version for free...
> > Why wouldn't we all encourage Nibre with some nice festive beer or two?
> > [Nibre's Paypal](https://www.paypal.me/NibreVR)
> > Merry Christmas and thanks for the hard work!
> 
> Good idea, done!

Excellent idea. Merry Christmas Nibre. Have a drink on me 👍🏻

**Comment by MartinNuc** [comment]
Yes this is the right attitude! Not asking every day when it's going to be done... 👏

✅ donated

**Comment by egena-its** [comment]
Just sent you a donation.  Your mod is beautiful, please poke it for the Epic version 😊

**Comment by djstatik** [comment]
Just donated as well.  This is going to be EPIC.  (pun intended)

**Comment by Bluscream** [comment]
![](https://i.imgur.com/6hZtTl3.png)

**Comment by mime29** [comment]
@Nibre can you just confirm that it is your Paypal please? 🥇

**Comment by MartinNuc** [comment]
You can check it on the 0.8 release: https://github.com/Nibre/MotherVR/releases/tag/0.8.0

**Comment by Nibre** [comment]
Thank you guys so much for the support! 😄

I've just pushed a new release that should now also work on the Epic Store version 
https://github.com/Nibre/MotherVR/releases

**Comment by DocJellyBean** [comment]
Thank you very much!

**Comment by hypercoyote** [comment]
Many thanks!

**Comment by MConstance05** [comment]
Thanks!, but how do i get it to work with virtual desktop? It gives me an error. Oculus failed to initialise, no hmd connected

**Comment by mime29** [comment]
Tested on HP Reverb G2, Epic Store version, working like a charm!
Amazing Job!

**Comment by PvtCowboy** [comment]
> Thanks!, but how do i get it to work with virtual desktop? It gives me an error. Oculus failed to initialise, no hmd connected

I had the same issue, I run virtual streamer from a cloud based pc (shadow tech) and steam over the Internet to my Oculus Quest 2.

To get around this problem I launched SteamVR first, from within there I selected desktops and then started AI. First time I ran AI with Mother Vr enabled the game had issues with the cut scene / loading, and one the game had loaded it was just a black screen (although I could bring up the map?) but I quit out and tried it again and had no issues. It works amazingly well, just remember to keep spare pants nearby!

**Comment by MConstance05** [comment]
> > Thanks!, but how do i get it to work with virtual desktop? It gives me an error. Oculus failed to initialise, no hmd connected
> 
> I had the same issue, I run virtual streamer from a cloud based pc (shadow tech) and steam over the Internet to my Oculus Quest 2.
> 
> To get around this problem I launched SteamVR first, from within there I selected desktops and then started AI. First time I ran AI with Mother Vr enabled the game had issues with the cut scene / loading, and one the game had loaded it was just a black screen (although I could bring up the map?) but I quit out and tried it again and had no issues. It works amazingly well, just remember to keep spare pants nearby! 

Yeah, thanks i have managed to get it working now!!

**Comment by djstatik** [comment]
Hey Nibre (and all), I was able to start the game with the new build.  Thanks Nibre!  The next day, I went to "Continue" and I can't find an input on the OC2 Touch controller that will allow me to continue.  I would think it would be "A".  I can start a new game using "A" without issue, but I cannot continue a game.  Please advise.

**Comment by AncorImparo** [comment]
I can confirm that the latest version MotherVR v0.81(?) as of this date, works fine with Alien Isolation Epic version with Oculus Quest 2. Do like this:

Start Virtual Desktop. Start SteamVR. Then you will land in this SteamVR room where you can access your Steam library and start the games. If you are not in this room, in the Win10 SteamVR small window, you right click the mouse and select "Display VR view" to enter the SteamVR room. 

In this SteamVR room, you can also popup a panel where you can do different stuff by pressing the left menu key (the key which is a hole on the left controller). Down to the left in the new panel you can choose "Desktops". If you press Desktop, you will see your Win10 screen. In this Win10 screen, you can use your mouse to select Alien Isolation (Epic version) and start it. For me it takes 1 minute, and then SteamVR says "Next up: Alien Isolation" and the game starts!
